README.md

```diff
@@ -1,54 +1,70 @@
+1. Arranging notebook outputs in a grid- or report-like layout
+2. Bundling notebooks and associated assets for deployment as dashboards
 3. Serving notebook-defined dashboards as standalone web apps
 
-This repository focuses on (3) above, while [jupyter-incubator/dashboards](https://github.com/jupyter-incubator/dashboards) handles (1) and [jupyter-incubator/dashboards_bundlers](https://github.com/jupyter-incubator/dashboards_bundlers) implements (2).
+This repository focuses on (3) above, while
+[jupyter-incubator/dashboards](https://github.com/jupyter-incubator/dashboards)
+handles (1) and
+[jupyter-incubator/dashboards_bundlers](https://github.com/jupyter-incubator/dashboards_bundlers)
+implements (2).
 
-See https://github.com/jupyter-incubator/dashboards/wiki for an overview of the entire dashboards effort.
+See https://github.com/jupyter-incubator/dashboards/wiki for an overview of the entire dashboard incubation effort.
 
 ## What it Gives You
 
-* Ability to run a Jupyter Notebook with [layout metadata](https://github.com/jupyter-incubator/dashboards) as a standalone dashboard application
-* Ability to navigate a list of multiple notebooks and select one to run as a dashboards
+* Ability to run **some** Jupyter notebooks as standalone dashboard applications
+* Ability to navigate a list of multiple notebooks and select one to run as a
+  dashboard
 * Optional shared login to secure access to the dashboard server
-* Ability to add custom authentication mechanisms using the [Passport](http://passportjs.org/) middleware for Node.js
-* API for POSTing notebooks to the server at runtime with optional authentication (`/_api/notebooks`)
+* Ability to add custom authentication mechanisms using the
+  [Passport](http://passportjs.org/) middleware for Node.js
+* An API for POSTing notebooks to the server at runtime with optional
+  authentication (`/_api/notebooks`)
 
-The behavior of the application is similar to that of [Thebe](https://github.com/oreillymedia/thebe), but with some key technical differences:
-
-* The notebook code is visible only in the NodeJS application backend.
-* The NodeJS backend is the only actor that can send notebook code to a kernel for execution.
-* The browser client can only send [Jupyter comm messages](http://jupyter-client.readthedocs.org/en/latest/messaging.html#opening-a-comm) to kernels (*not* arbitrary code).
-* The application uses the [jupyter-js-services](https://github.com/jupyter/jupyter-js-services) and [jupyter-js-widgets](https://github.com/ipython/ipywidgets/tree/master/jupyter-js-widgets) libraries for communication with kernels.
-
-The following libraries are known to work with the dashboard server:
+The qualification in the first bullet stems from the fact that supporting
+one-click deploy of notebooks with arbitrary JavaScript and kernel dependencies
+is a "Really Hard Problem." We've invested effort in getting these dashboard,
+visualization, and widget libraries working in the dashboard server.
```

Comment:
Instead of calling out the "first bullet", can you instead use footnote-style notation (i.e. "[1]" at end of bullet and same string at start of this sentence)? I feel that that will be more maintainable, in case the bullet list above is reshuffled/changed.